pkgs/by-name/cz/czkawka/package.nix

```diff
@@ -1,22 +1,33 @@
 {
   lib,
+  stdenv,
+  rustPlatform,
+  overrideSDK,
+  fetchFromGitHub,
+
+  # nativeBuildInputs
+  gobject-introspection,
+  pkg-config,
+  wrapGAppsHook4,
+
+  # buildInputs
   atk,
   cairo,
-  callPackage,
-  darwin,
-  fetchFromGitHub,
   gdk-pixbuf,
   glib,
-  gobject-introspection,
   gtk4,
-  overrideSDK,
   pango,
-  pkg-config,
-  rustPlatform,
-  stdenv,
-  testers,
-  wrapGAppsHook4,
+  darwin,
+
+  # nativeCheckInputs
   xvfb-run,
+
+  # nativeInstallCheckInputs
+  versionCheckHook,
+
+  # passthru
```

Comment:
Why such redundancy?

Reply:
I personally find it clearer like this. Of course if you do not share this opinion I can promptly remove it.

Reply:
I don't think this is useful, given that the same info is written in the `stdenv.mkDerivation` - and with the clear advantage of being executable code.
Please remove it.

Reply:
Sure, done.